how-to/about-how-to.md

```diff
@@ -40,7 +40,7 @@ Make sure you meet the following prerequisites before following the steps:
 
 * API credentials for the {company} v3.5 API.
 * Access to the Postman application.
-* An understanding about what a RESTful API is conceptually.
+* An understanding about what a RESTful API is, conceptually.
```

Comment:
What do you think about swapping out the "about" with an "of":

An understanding of what a RESTful API is, conceptually.

Reply:
Sure; seems good to me, too.

Reply:
Will you make this change and push another revision @Dishebh ?

Reply:
I already committed the change. Please check out my commit with the commit message of "minor update of swapping out 'about' with an 'of'".